parts/k8s/artifacts/kuberneteskubelet.service

```diff
@@ -30,14 +30,15 @@ ExecStart=/usr/bin/docker run \
   --volume=/etc/kubernetes/:/etc/kubernetes:ro \
   --volume=/srv/kubernetes/:/srv/kubernetes:ro $DOCKER_OPTS \
   --volume=/var/lib/waagent/ManagedIdentity-Settings:/var/lib/waagent/ManagedIdentity-Settings:ro \
-  --volume=/usr/libexec/kubernetes/kubelet-plugins:/usr/libexec/kubernetes/kubelet-plugins:rw \
+  --volume=/etc/kubernetes/volumeplugins:/etc/kubernetes/volumeplugins:rw \
```

Comment:
does this need to be done for the 1.5 kuberneteskubelet.service as well?

Reply:
(I mean in parts/k8s/artifacts/1.5/kuberneteskubelet.service)

Reply:
@anhowe k8s 1.5 does not support flexvolume, so it's not necessary